libblkid/src/superblocks/exfat.c

```diff
@@ -161,6 +161,8 @@ static int exfat_validate_checksum(blkid_probe pr,
 	return 1;
 }
 
+#define in_range_inclusive(val, start, stop) (val >= start && val <= stop)
+
```

Comment:
What about to adding in_range() to include/c.h? There are already macros to compare numbers.